test/test.js

```diff
@@ -756,3 +760,21 @@ test('display test title prefixes', function (t) {
 		t.is(tests.length, 0);
 	});
 });
+
+test('fail-fast mode', function (t) {
+	t.plan(5);
+
+	execCli(['fixture/fail-fast.js', '--fail-fast'], function (err, stdout, stderr) {
+		t.ok(err);
+		t.is(err.code, 1);
+
+		var output = stderr
+			.replace(new RegExp(figures.tick, 'gm'), 'success')
+			.replace(new RegExp(figures.cross, 'gm'), 'error');
```

Comment:
Instead of doing a replace here just put the `figures` directly into the `indexOf` check.